dev/plot_film_parameters.py

```diff
@@ -22,40 +25,46 @@ def plot_histogram(data, layer_no, fname_out):
     :param layer_no: number of the layer to consider
     :param fname_out: directory to save the figure
     """
-    bins = np.linspace(-10, 10, 100)
+    bins = np.linspace(0, 1, 100)
```

Comment:
I thought we may expect negative values. Could you help me here?

Reply:
Finally I kept the original FiLM implementation that restricts values from 0 to 1 with the sigmoid, it also gave me better results on my test dataset.